airflow/providers/samba/hooks/samba.py

```diff
@@ -245,10 +245,22 @@ def setxattr(self, path, attribute, value, flags=0, follow_symlinks=True):
             **self._conn_kwargs,
         )
 
-    def push_from_local(self, destination_filepath: str, local_filepath: str):
-        """Push local file to samba server."""
-        with open(local_filepath, "rb") as f, self.open_file(destination_filepath, mode="wb") as g:
-            copyfileobj(f, g)
+    def push_from_local(self, destination_filepath: str, local_filepath: str, buffer_len: int | None = None):
+        """
+        Push local file to samba server.
+
+        :param destination_filepath: the samba location to push to
+        :param local_filepath: the file to push
+        :param buffer_len:
```

Comment:
Small nit, maybe call it `buffer_size` or `buffer_length` ?

Reply:
I'll switch to `buffer_size`